superset/viz.py

```diff
@@ -2765,6 +2765,15 @@ def get_data(self, df):
         return self.nest_values(levels)
 
 
+class EchartsScatter(BubbleViz):
```

Comment:
Could you use the new api instead?

Reply:
@kristw is there already a working plan for migrating some of the more complex legacy viz's to the new api, namely the ones that rely on Pandas functionality? Is the plan to do that logic in JS, och making the heavy lifting in the backend? I anticipate quite a bit of work there, and can lend a hand if necessary, especially if the new API needs new functionality..

Reply:
We had light discussions about listing `postprocessing` operations as a new field in the `api/v1/query` parameters to call those pandas functionality, but none had the cycle to draft the API spec nor work on it yet. If you would like to step in, that could be helpful.

Reply:
Sounds reasonable. In all, I think this will be a big feature to implement, but can probably be dissected into smaller parts. I would probably start with implementing the pivot operation, as it is used quite frequently in viz's and should be easy to start with. I'm fairly overloaded with other stuff right now, but should be able to look into this in a few weeks time if nobody wants to take the lead.

Reply:
Has anyone looked at this? Not sure if it is suitable here https://github.com/nickslevine/zebras

Reply:
It also doesn't have to be either done in the frontend or backend; can be both. Some postprocessing can be very expensive to move to the front, especially percentiles/quantiles and the like that require having a full distribution, so those would be preferable to perform in the backend, while others might be well suited for the frontend. So I think being able to offer at least some postprocessing options in the backend would be nice, leaving it up to the viz plugin developer to decide where to perform the calculations.